fdbserver/storageserver.actor.cpp

```diff
@@ -13682,14 +14009,15 @@ ACTOR Future<Void> storageServerCore(StorageServer* self, StorageServerInterface
 					self->auditTasks[req.getType()] = std::make_pair(req.id, ActorCollection(true));
 				}
 				// Start the new audit task
+				ASSERT(req.ddId.isValid()); // ddId is used when persist progress
```

Comment:
Beging paranoid, is it possible to fail the audit request, instead of crash the server due to an error from DD?

Reply:
Yes. Thanks for suggestion!